src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/LocationResidentialResource.kt

```diff
@@ -385,7 +385,7 @@ class LocationResidentialResource(
     @field:Max(value = 99, message = "Working capacity cannot be greater than 99")
     @field:PositiveOrZero(message = "Working capacity cannot be less than 0")
     val workingCapacity: Int = 0,
-    @Schema(description = "Used For list", example = "STANDARD_ACCOMMODATION, PERSONALITY_DISORDER", required = false)
+    @Schema(description = "Used For list", example = "[ STANDARD_ACCOMMODATION, PERSONALITY_DISORDER ]", required = false)
```

Comment:
Should this be `"[ \"STANDARD_ACCOMMODATION\", \"PERSONALITY_DISORDER\" ]"`?

Reply:
Yes!

Reply:
@jimbali re-submitted